routes/routes.js

```diff
@@ -44,7 +44,13 @@ const appRouter = function (app, gulp) {
       const prebidDownloadFilePath = path.resolve(prebidPath, 'build/dist', fileName);
       console.log(output);
       if(fs.existsSync(prebidDownloadFilePath)) {
-        res.download(prebidDownloadFilePath);
+        res.download(prebidDownloadFilePath, function (err) {
+          if (err) {
+            res.end();
+          } else {
+            fs.unlinkSync(prebidDownloadFilePath);
```

Comment:
This looks good, the only change I would make would be to have this be an async operations (no reason to hang the client while a file is deleted).  So switch to `fs.unlink`.

Reply:
Will do, thanks!